Bottom: 9ceee3ed5d3b8307cdf04d2f0cca8313992ef22e
Top:    a382695e2987cbd0d5a442f0e357413683b8c1df
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-08-16 14:26:03 -0700

Refresh of waitall-buggy

---

diff --git a/src/rsocket.c b/src/rsocket.c
index e48c305..2edd44e 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -1141,7 +1141,7 @@ ssize_t rrecv(int socket, void *buf, size_t len, int flags)
 
 		ret = 0;
 		if (flags & MSG_PEEK) {
-			left = len - rs_peek(rs, &buf, left);
+			left = len - rs_peek(rs, buf, left);
 			break;
 		}
